source/content.js

```diff
@@ -171,6 +170,8 @@ function ajaxedPagesHandler() {
 		enableFeature(addCommentedMenuItem);
 	}
 
+	enableFeature(sortIssuesByUpdateTime); // Must be before addYoursMenuItem + addCommentedMenuItem
+
```

Comment:
@bfred-it you sure the comment is correct here? I see you moved `sortIssuesByUpdateTime` **after** the `addYoursMenuItem` + `addCommentedMenuItem` but the comment says _**before**_ :smile: 

Or I might have misunderstood..

Reply:
Oops.